webapp/pages/profile/_id/_slug.vue

```diff
@@ -417,6 +420,24 @@ export default {
         this.$apollo.queries.profilePagePosts.refetch()
       }
     },
+    async blockUser(user) {
+      try {
+        await this.$apollo.mutate({ mutation: blockUser(), variables: { id: user.id } })
```

Comment:
```suggestion
        await this.$apollo.mutate({ mutation: blockUser(), variables: user })
```
Would be OK, too, I guess. I'm OK to only pass the `id` call though. Choose whatever you like more. Maybe the explicit `id` is more, well, explicit.

Reply:
I copied what you added when you introduced the block feature... and I guess it's good enough ;)